doc/sphinx/proof-engine/ltac.rst

```diff
@@ -988,6 +1167,11 @@ Pattern matching on terms: match
       into :n:`@ltac_expr` before it's applied.  Note that metavariables are not
       prefixed with the question mark as they are in :token:`cpattern`.
 
+      If :token:`ltac_expr` evaluates to a tactic, then it is
+      applied. If the tactic succeeds, the list of resulting subgoals is the
+      result of the match expression. If :token:`ltac_expr` does not evaluate
```

Comment:
```suggestion
      applied. If the tactic succeeds, the unit value of :tacn:`idtac` is the
      result of the match expression. If :token:`ltac_expr` does not evaluate
```

Reply:
Meaning it it is the no-op tactic--not another type of value as is in OCaml, right?  Maybe don't need to say "unit value"?

Reply:
Yes, that's correct.  (There is a separate value of unit type, denoted `()`, which is not valid as a tactic to be run on the goal, and the result is not that value.)  I'd also be fine with 
```suggestion
      applied. If the tactic succeeds, the tactic value :tacn:`idtac` is the
      result of the match expression. If :token:`ltac_expr` does not evaluate
```
or
```suggestion
      applied. If the tactic succeeds, the tactic :tacn:`idtac` is the
      result of the match expression. If :token:`ltac_expr` does not evaluate
```
or

```suggestion
      applied. If the tactic succeeds, the result of the match expression
      is :tacn:`idtac`. If :token:`ltac_expr` does not evaluate
```